mw-webapp/src/styles/_variables.scss

```diff
@@ -10,6 +10,7 @@ $link-active: rgb(255, 149, 0);
 
 // Header
 $header-background: rgb(85, 122, 149);
+$header-padding: 0 1em;
```

Comment:
Why padding? It's looks uncentered

Reply:
and how it should be?

Reply:
removed the extra padding from the title, thanks for noticing